sdk/core/core-rest-pipeline/src/util/userAgentPlatform-react-native.mts

```diff
@@ -1,9 +1,7 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-/*
- * NOTE: When moving this file, please update "react-native" section in package.json.
- */
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
```

Comment:
I wonder if we could forward-declare the module (or have a dummy .d.ts next to this file that declares it) to avoid needing the ts-ignore here

Reply:
Adding a `.d.ts` file works!